src/ring_theory/polynomial/basic.lean

```diff
@@ -670,11 +670,53 @@ lemma is_domain_map_C_quotient {P : ideal R} (H : is_prime P) :
 ring_equiv.is_domain (polynomial (R ⧸ P))
```

Comment:
I assume this reference to quotient rings gets moved in a later commit?

Reply:
Yes.